gapir/cc/memory_manager.cpp

```diff
@@ -22,74 +22,42 @@
 #include <utility>
 #include <vector>
 
-//                 mMemory[0]
-//   ┏━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━┓
-//   ┃             ┃                        ┃
-//   ┃             ┃                        ┃
-//   ┃             ┃                        ┃
-//   ┃             ┃         volatile       ┃
-//   ┃   mMemory   ┃          memory        ┃
-//   ┃             ┃                        ┃
-//   ┃             ┃                        ┨
-//   ┃             ┣━━━━━━━━━━━━━━━━━━━━━━━━┨
-//   ┃             ┃                        ┃
-//   ┃             ┃        in-memory       ┃
-//   ┃             ┃        resource        ┃
-//   ┃             ┃          cache         ┃
-//   ┃             ┃                        ┃
-//   ┗━━━━━━━━━━━━━┻━━━━━━━━━━━━━━━━━━━━━━━━┛
-//                 mMemory[mSize]
-
 namespace gapir {
 
-namespace {
-// Expected driver memory overhead to be left free as a factor of allocated
-// managed memory.
-const float kDriverOverheadFactor = 0.3f;
-}  // namespace
-
 template <typename T>
 MemoryManager::MemoryRange<T>::MemoryRange() : base(nullptr), size(0) {}
 
 template <typename T>
 MemoryManager::MemoryRange<T>::MemoryRange(T* base, uint32_t size)
     : base(base), size(size) {}
 
-MemoryManager::MemoryManager(const std::vector<uint32_t>& sizeList)
-    : mConstantMemory(nullptr, 0) {
-  for (auto size : sizeList) {
```

Comment:
We probably do want to keep this behavior even if it we end up having to move it. It means we wont starve other tools.

Reply:
Addressed. Check the allocator construction in main.cpp.